index.html

```diff
@@ -10,23 +10,215 @@
 
 <body class="body">
 
-  <div class="navbar">
-    <div class="my-logo">My Logo</div>
-    <a class="menu-icon"></a>
-  </div>
-
-  <div class="main">
-    <div class="primary-text">I'm Juan Diaz Glad to see you!</div>
-    <div class="secondary-text">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</div>
+  <section class="intro">
+    <header>
+      <nav class="navbar">
+        <a class="my-logo">My Logo</a>
+        <a class="menu-icon"></a>
+      </nav>
+    </header>
+
+
+    <div class="main">
+      <div class="primary-text-container">
+        <h1 class="primary-text">I'm Juan Diaz Glad to see you!</h1>
+      </div>
+      <div class="secondary-text">
+        <p>I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      </div>
+      <div class="connect-text">LET'S CONNECT</div>
+      <div class="socialmedia-icons">
+        <ul>
+          <li class="twitter-icon"><a href=""></a></li>
+          <li class="linkedin-icon"><a href=""></a></li>
+          <li class="m-icon"><a href=""></a></li>
+          <li class="cat-icon"><a href=""></a></li>
+          <li class="fingers-icon"><a href=""></a></li>
+        </ul>
+      </div>
+    </div>
+  </section>
+
+  <section class="projects">
+    <div class="project" id="project1">
+      <div class="pimage"></div>
+      <div class="p-title">
+        <h2>Tonic</h2>
+      </div>
+      <div class="projectdetails">
+        <ul class="details-list">
+          <li class="client">CANOPY</li>
+          <li class="spacedot"></li>
+          <li class="role">Back End Dev</li>
+          <li class="spacedot"></li>
+          <li class="year">2015</li>
+        </ul>
+      </div>
+      <div class="p-description">
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      </div>
+      <div class="languages">
+        <ul class="languages-list">
+          <li class="lang">html</li>
+          <li class="lang">css</li>
+          <li class="lang">javaScript</li>
+        </ul>
+      </div>
+      <div class="project-access">See project</div>
```

Comment:
- [ ] I think it would be better to use `<button>` or `<a>` tag here "See project" instead of `div` tag. because this is a link 👌. And please don't forget to follow the button styling: Active, Hover, and Pressed.

![2](https://user-images.githubusercontent.com/45231276/176837061-facdfea9-a844-49bc-bed7-93be8ab7b49c.PNG)

Reply:
Now using the 'button'. Appreciate the feedback!